bot/exts/moderation/infraction/_utils.py

```diff
@@ -31,8 +30,10 @@
 
 APPEAL_EMAIL = "appeals@pythondiscord.com"
 
-INFRACTION_TITLE = f"Please review our rules over at {RULES_URL}"
-INFRACTION_APPEAL_FOOTER = f"To appeal this infraction, send an e-mail to {APPEAL_EMAIL}"
+INFRACTION_TITLE = "Please review our rules"
+INFRACTION_APPEAL_EMAIL_FOOTER = f"To appeal this infraction, send an e-mail to {APPEAL_EMAIL}"
+INFRACTION_APPEAL_MODMAIL_FOOTER = ('If you would like to discuss or appeal this infraction, '
+                                    'send a message to the ModMail bot')
```

Comment:
Could we make it a mention of the modmail bot? Changing `ModMail` to `<@683001325440860340>` should do it.

Reply:
I tried this and it didn't render as expected, presumably because the DM is out-of-guild content.

Reply:
Hmm yeah it can cause issues if it isn't in the cache. Could we maybe have both, like having a mention in parenthesis along with the bot name?

Reply:
I don't think that would improve the footer, as it's already long as it is. @Akarys42 thoughts?

Reply:
I’m fine with this current text :+1:

Reply:
I don't think it's worth risking the footer being uninformative (I think it would just say `invalid-user`) if the bot, for some reason, weren't in their cache.

Reply:
Personally, it seems far too wide. At the very least, add a `\n` character.